install-required-packages.sh

```diff
@@ -613,7 +613,7 @@ declare -A pkg_python_mysqldb=(
 	 ['centos']="MySQL-python"
 	 ['debian']="python-mysqldb"
 	 ['gentoo']="dev-python/mysqlclient"
-	   ['rhel']="python-mysql"
+	   ['rhel']="MySQL-python"
```

Comment:
There is no `python-mysql` package available in either RHEL 6 or 7. The correct package name is `MySQL-python`